src/test/java/com/faforever/client/chat/PrivateChatTabControllerTest.java

```diff
@@ -118,18 +124,23 @@ public void setUp() throws Exception {
       }
       return instance;
     });
+  }
 
+  private void setReceiver() {
     instance.setReceiver(playerName);
-    WaitForAsyncUtils.asyncFx(() -> {
+    runOnFxThreadAndWait(() -> {
+      TabPane tabPane = new TabPane();
+      tabPane.setSkin(new TabPaneSkin(tabPane));
       getRoot().getChildren().setAll(tabPane);
       tabPane.getTabs().add(instance.getRoot());
-    }).get();
-
+    });
     verify(webViewConfigurer).configureWebView(eq(instance.messagesWebView));
   }
 
   @Test
   public void testOnChatMessageUnfocusedTriggersNotification() {
+    setReceiver();
```

Comment:
Can setReceiver just be added to the setup function?

Reply:
Can. But then the `checkTabHasNoAvatar` test will fail.

Reply:
In that case you could just set the avatar to null and call setReceiver again

Reply:
Thanks for explain